sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -55,9 +56,9 @@
   - Read alert_info from rstmgr and compare with expected value.
 
   Round 3: All Classes profile with alert escalation
-  - Trigger local alert by alert handler ping timeout
+  - Trigger rv_core_ibex alert that leads to an interrupt.
```

Comment:
Tim, appreciate your hard work to make this test compatible to both platforms.
Ping timeout had two purposes. one is to trigger interrupt and it is ok to replace with the other
like this one.

Another purpose was to create 'local' alert, which is also part of alert_info field.
I am not suggesting to update 'this' test, since this test is complicated enough.
But we need another separate test to create local and global alert to see, alert handler can
capture both events properly and send this to rsgmgr.
LMK, I can create task and tag here for the reference.

Reply:
Yeah I think maybe using a different test to force alert not response to a ping might be a better idea to check local alerts.
The current method will trigger some spurious alert: https://github.com/lowRISC/opentitan/issues/2321